app/models/bus_stop.rb

```diff
@@ -6,6 +6,7 @@ class BusStop < ApplicationRecord
   has_paper_trail
   validates :name, presence: true
   validates :hastus_id, presence: true, uniqueness: true
+  validates :check_completed, presence: { if: :completed? }
```

Comment:
To use a custom validation method, you want `validate`, not `validates`, correct?

I think it would be good to see a unit test in any case.